views/partials/_example.njk

```diff
@@ -32,13 +32,17 @@
     <li class="app-tabs__item js-tabs__item{{ " js-tabs__item--open" if (params.open) }}" role="presentation"><a href="#{{ exampleId }}-html" role="tab" aria-controls="{{ exampleId }}-html" data-track="tab-html">HTML</a></li>
     <li class="app-tabs__item js-tabs__item" role="presentation"><a href="#{{ exampleId }}-nunjucks" role="tab" aria-controls="{{ exampleId }}-nunjucks" data-track="tab-nunjucks">Nunjucks</a></li>
   </ul>
+  {% elif not (multipleTabs) and (not params.hideTab) %}
+  <ul class="app-tabs" role="tablist">
+    <li class="app-tabs__item js-tabs__item{{ " js-tabs__item--open" if (params.open) }}" role="presentation">
+      <a href="#{{ exampleId }}-{{ "html" if params.html else ("nunjucks" if params.nunjucks )}}" role="tab" aria-controls="{{ exampleId }}-{{ "html" if params.html else ("nunjucks" if params.nunjucks )}}" data-track="tab-{{ "html" if params.html else ("nunjucks" if params.nunjucks )}}">{{ "HTML" if params.html else ("Nunjucks" if params.nunjucks )}}</a>
+    </li>
+  </ul>
```

Comment:
Not blocking but there is some repetition of code here, I wonder if the `if` and `elif` blocks could be combined to avoid some of that.

Reply:
abstracted to a variable and use that inplace, apart for the text where we do diffent treatment for HTML and Nunjucks